pkgs/by-name/ti/ticktick/package.nix

```diff
@@ -71,8 +85,11 @@ stdenv.mkDerivation (finalAttrs: {
     description = "Powerful to-do & task management app with seamless cloud synchronization across all your devices";
     homepage = "https://ticktick.com/home/";
     license = licenses.unfree;
-    maintainers = with maintainers; [ hbjydev ];
-    platforms = [ "x86_64-linux" ];
+    maintainers = with maintainers; [
+      hbjydev
+      jonocodes
```

Comment:
You need to add yourself to the maintainers file as part of this PR too, this fails builds currently.